src/Common/src/TypeSystem/Common/MetadataFieldLayoutAlgorithm.cs

```diff
@@ -346,7 +349,7 @@ private static ComputedInstanceFieldLayout ComputeExplicitFieldLayout(MetadataTy
             return computedLayout;
         }
 
-        private static ComputedInstanceFieldLayout ComputeSequentialFieldLayout(MetadataType type, int numInstanceFields)
+        protected virtual ComputedInstanceFieldLayout ComputeSequentialFieldLayout(MetadataType type, int numInstanceFields)
```

Comment:
Why does this method need to be virtual? I don't see any overrides in your change. After all, while I can easily imagine there may be multiple ways to lay out the class fields, I would naively assume there's just one sequential layout.

Reply:
Ah you're right, this doesn't need to be marked virtual. I accidentally marked this one virtual because that's how I marked ComputeAutoFieldLayout. If I get approval on this class design I'll go back and remove the virtual keyword. Thanks for pointing this out